visualization/app/codeCharta/ui/codeMap/rendering/geometryGenerator.ts

```diff
@@ -252,7 +186,7 @@ export class GeometryGenerator {
 	private createAndAssignFloorLabelTextureMaterial(surfaceInfo: SurfaceInformation) {
```

Comment:
Parameter names (height and width) used for calculation are contradictory to x, y and z.